net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Input.xml

```diff
@@ -93,5 +93,9 @@
             <default>0</default>
             <Required>N</Required>
         </intrusion_detection_alerts>
+        <unbound type="BooleanField">
+            <default>0</default>
```

Comment:
I don't want to set the practice to inject default values to non-required (empty) vars.

```suggestion
```